src/SprykerEco/Yves/Computop/ComputopFactory.php

```diff
@@ -555,4 +654,12 @@ public function getCountryClient(): ComputopToCountryClientInterface
     {
         return $this->getProvidedDependency(ComputopDependencyProvider::CLIENT_COUNTRY);
     }
+
+    /**
+     * @return \SprykerEco\Yves\Computop\Mapper\Init\PrePlace\PayPalExpressToQuoteMapper
+     */
+    protected function createPayPalExpressToQuoteMapper(): PayPalExpressToQuoteMapper
```

Comment:
```suggestion
    protected function createPayPalExpressToQuoteMapper(): PayPalExpressToQuoteMapperInterface
```

please use interfaces where it's possible